hugo/content/install-setup/aks.md

```diff
@@ -0,0 +1,193 @@
+
+
+```plain
+export AZ_RESOURCE_GROUP=<myResourceGroup>
+export AZ_RESOURCE_GROUP_LOCATION=eastus
+```
+
+{{< copyable name="step-01" >}}
+az group create --name $AZ_RESOURCE_GROUP --location $AZ_RESOURCE_GROUP_LOCATION
+{{< /copyable >}}
+
+## Provision cluster
+
+Then we provision a new Kubernetes cluster of 3 nodes:
+
+{{< copyable name="step-02" >}}
+az aks create --resource-group $AZ_RESOURCE_GROUP --name myAKSCluster --node-count 3 --generate-ssh-keys --kubernetes-version 1.8.7
+{{< /copyable >}}
+
+
+## Create namespace & secrets
+
+Before we can install Dotmesh, we need to set our admin password and api key:
+
+```plain
+export ADMIN_PASSWORD=apples
+export ADMIN_API_KEY=apples
+```
+
+Then we create the namespace before adding our credentials as secrets:
+
+{{< copyable name="step-04" >}}
+kubectl create namespace dotmesh
+echo -n $ADMIN_PASSWORD > dotmesh-admin-password.txt
+echo -n $ADMIN_API_KEY > dotmesh-api-key.txt
+kubectl create secret generic dotmesh \
+  --from-file=./dotmesh-admin-password.txt \
+  --from-file=./dotmesh-api-key.txt -n dotmesh
+rm -f dotmesh-admin-password.txt dotmesh-api-key.txt
+{{< /copyable >}}
+
+## Etcd operator
+
+Install the etcd operator ready for our dotmesh cluster:
+
+{{< copyable name="step-05" >}}
+kubectl apply -f https://get.dotmesh.io/yaml/etcd-operator-clusterrole.yaml
+kubectl apply -f https://get.dotmesh.io/yaml/etcd-operator-dep.yaml
+{{< /copyable >}}
+
+It may take a few minutes for the etcd operator to activate.
+Use `kubectl get pods -n dotmesh` to check for a running `etcd-operator` pod.
+
+## Create an etcd cluster for dotmesh to use
+
+{{< copyable name="step-05a" >}}
+kubectl apply -f https://get.dotmesh.io/yaml/dotmesh-etcd-cluster.yaml
+{{< /copyable >}}
+
+```plain
+etcdcluster "dotmesh-etcd-cluster" configured
+```
+
+## Dotmesh
+
+Use the following command to apply the YAML configuration for running dotmesh:
+
+{{< copyable name="step-06" >}}
+kubectl apply -f https://get.dotmesh.io/yaml/dotmesh-k8s-1.8.aks.yaml
+{{< /copyable >}}
+
+```plain
+serviceaccount "dotmesh" configured
+clusterrole "dotmesh" configured
+clusterrolebinding "dotmesh" configured
+service "dotmesh" configured
+daemonset "dotmesh" configured
+serviceaccount "dotmesh-provisioner" configured
+clusterrole "dotmesh-provisioner-runner" configured
+clusterrolebinding "dotmesh-provisioner" configured
+deployment "dotmesh-dynamic-provisioner" configured
+storageclass "dotmesh" configured
+```
+
+**NOTE** - This will create a service of type Loadbalancer that will expose port port 6969 on the Internet. The need for a firewall rule will be replaced with an ingress rule in an upcoming release.
```

Comment:
I've updated the generate manifest from service type ClusterIP to service type Loadbalancer. I assume this is okay as the GKE steps essentially do the same thing via a NodePort. See corresponding sed expression in https://github.com/dotmesh-io/dotmesh/pull/315